Index: src/main/java/bds.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/bds.java	(date 1466981798000)
+++ src/main/java/bds.java	(revision )
@@ -106,7 +106,7 @@
 
     public void a(ey var, int var1) {
         this.a(var, var1, this.h.r.e(), false);
-        g.info("[CHAT] {}", new Object[]{f.join(a.split(var.c()))});
+        g.info("[CHAT] {}", new Object[]{f.join(a.split(var.d()))});
     }
 
     private void a(ey var, int var1, int var2, boolean var3) {
