Index: src/main/java/buh.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/buh.java	(date 1444672737000)
+++ src/main/java/buh.java	(revision )
@@ -101,7 +101,7 @@
 
     public void a(ho var1, int var2) {
         this.a(var1, var2, this.f.q.e(), false);
-        a.info("[CHAT] " + var1.c());
+        a.info("[CHAT] " + var1.d());
     }
 
     private void a(ho var1, int var2, int var3, boolean var4) {
